src/fu-engine.c

```diff
@@ -2006,6 +2006,28 @@ fu_engine_get_boot_time(void)
 	return NULL;
 }
 
+static void
+fu_engine_get_report_metadata_cpu_device(FuEngine *self, GHashTable *hash)
+{
+	g_autofree gchar *guid = fwupd_guid_hash_string("cpu");
```

Comment:
This feels like it's getting too specifically into the current situation of some older CPUs don't export the needed information.

I feel we should be sticking to whether ANY plugin doesn't export enough information to better scale for future measurements.

Reply:
This is getting the device with that attribute, not the security attribute. i.e. it's the data from CPUID.

Reply:
Oh right okay. No worry then.